tools/cli/go-whisk-cli/commands/api.go

```diff
@@ -256,37 +259,35 @@ var apiDeleteCmd = &cobra.Command{
             return whiskErr
         }
 
-        api := new(whisk.Api)
-        options := new(whisk.ApiOptions)
-        options.Force = true
-
+        apiDeleteReq := new(whisk.ApiDeleteRequest)
+        apiDeleteReqOptions := new(whisk.ApiDeleteRequestOptions)
         // Is the argument a basepath (must start with /) or an API name
         if _, ok := isValidBasepath(args[0]); !ok {
```

Comment:
Why is this check necessary when the if and else paths set the same variable to args[0]?